src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,336 @@
+
+  When an incident closes automatically:
+
+  1. The closing timestamp is backdated to the start of the recovery period.
+  2. The evaluation resets and restarts from when the previous incident ended.
+
+  All conditions have an incident time limit setting that will automatically force-close a long-lasting incident.
+
+  We automatically default to 3 days and recommend that you use our default settings for your first alert. 
+  </Collapser>
+
+  <Collapser
+    id="custom-incident-description"
+    title="Send a custom incident description"
+  >
+  
+  Since we're creating an alert condition that lets us know if there are any latency issues with our `WebPortal` application, we want to make sure our developers have all the information they need when they are notified about this incident. We're going to use workflows to notify a team Slack channel when an incident is created.
+
+  Learn more about custom incident descriptions in our [docs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-incident-descriptions).
+
+</Collapser>
+
+  <Collapser
+    id="runbook"
+    title="Add runbook url"
+  >
+If you'd like to link to a runbook, you can put the URL in the runbook URL field.
+  </Collapser>
+</CollapserGroup>
+
+</Step>
+</Steps>
+
+## Edit or improve an existing alert condition [#edit-existing-alert-condition]
+
+If you want to edit an alert condition you've already created, you can:
+
+1. Go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**.
+2. Select **Alert Conditions** in the left navigation.
+3. Click on the alert condition you want to edit.
+
+From there, you will be able to see all of the elements you set when creating your condition. You can edit specific elements of the alert condition by clicking the pencil in top right of each section. 
+
+If you have questions about how to edit each section, refer back to `Create a new alert condition` section:
+
+* [Signal coverage](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#create-alert-condition)
+* [Signal behavior](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#advanced-signal-settings)
+* [Thresholds](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#thresholds)
+* [Additional details](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#additional-details) 
+
+### Signal Coverage [#signal-coverage]
+
+Signal coverage gives a closer look at the most recent signals being ingested by the condition. For Synthetics type conditions, a chart is shown with the 5 most recent `SyntheticChecks`. For all other types, a line chart showing the `NraiSignal` over time for the last few days is shown for easier analysis. This chart is similar to what is shown for an individual issue or incident.
```

Comment:
Signal history displays the activity of the signals that the condition has been producing.

Reply:
@corinnekunze when you say signal, can you give me a concrete example? I just want to be able to better explain this in plain language.